client/src/app/pages/issues/issues.tsx

```diff
@@ -300,9 +304,9 @@ export const Issues: React.FC = () => {
                                   <Tooltip content="View Report">
                                     <Button variant="link" isInline>
                                       <Link
-                                        // to={`/hub/applications/${application.id}/bucket${task?.data?.output}`}
-                                        to={""}
-                                        target="_blank"
+                                        // to={`${ANALYSIS_ISSUES}/${issue.ruleID}`}
+                                        to={`/issues/${issue.ruleID}`}
```

Comment:
@mturley I am wondering if you have a better idea of how to persist the filter state down to the affected apps page. This works, but probably need something that also preserves the URL bar so that we can still refresh the browser and remain on the affected apps page. Otherwise may just redirect back to the composite issues page if this isn't feasible.

Reply:
https://www.npmjs.com/package/query-string Looking into this as a solution for parsing the rule-id and just keeping the linked url as 

```
to={`/issues/?filter=ruleid%7E%22*${issue.ruleID}*%22&sort=asc:ruleID&limit=10%offset=0`}
```
to satisfy the server requests

Reply:
Query-string ended up not working out since we are doing some custom filtering. RegEx works fine, though.